pkg/api/v1/conversion.go

```diff
@@ -223,3 +225,136 @@ func convert_v1_ReplicationControllerSpec_To_api_ReplicationControllerSpec(in *R
 	}
 	return nil
 }
+
+// The following two PodSpec conversions are done here to support ServiceAccount
+// as an alias for ServiceAccountName.
+func convert_api_PodSpec_To_v1_PodSpec(in *api.PodSpec, out *PodSpec, s conversion.Scope) error {
```

Comment:
Didn't work :-( Doesn't appear to play well with defaults. Panics for some flags settings, and when specifying explicitly, hit "defaults_test.go:390: Expected container port to be defaulted, was made 0 instead of 8080".